.github/workflows/ci.yml

```diff
@@ -25,6 +25,13 @@ jobs:
       - name: Install test dependencies
         run: pip install ansible-lint[community,yamllint]
 
+      - name: Install roles required for tests
+        run: |
+          for req in $(find ./molecule/ -name 'requirements.yml'); do
+            echo Requirements file: $req
+            ansible-galaxy install -r $req
+          done
+
```

Comment:
Linter fails without installation of roles required in the molecule tests ([example](https://github.com/Appsilon/ansible-rstudio-connect/runs/8264767776?check_suite_focus=true))

Reply:
Both molecule and ansible-lint should install requirements automatically if there is requirements.yml file. I don't know why we have to do this manually. Can you explain a little bit more?

> In order to help Ansible find used modules and roles, molecule will perform a prerun set of actions. These involve installing dependencies from requirements.yml specified at project level, install a standalone role or a collection. 

Source: https://molecule.readthedocs.io/en/latest/configuration.html#id1

Reply:
> These involve installing dependencies from requirements.yml specified at project level

I thought that roles specified in requirements.yml file are always installed during installation (of the role - `appsilon.ansible-rstudio-connect`). `appsilon.python_install` is not required by this role, so I thought it may me wrong to put it there.

However, I've just checked that role's dependencies should be specified in `meta/requirements.yml` or `meta/main.yml` ([source](https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#dependencies)). So it shouldn't be an issue.

I will create requirements.yml with `appsilon.python_install` and update workflow definition.

Thanks for pointing this out!

Reply:
That's awesome you found that! Good job!

Reply:
Example:
<img width="590" alt="image" src="https://user-images.githubusercontent.com/46781099/190603553-f650fbcf-897e-4aad-bf80-b338edc86162.png">

Why not here?
<img width="593" alt="image" src="https://user-images.githubusercontent.com/46781099/190603696-ce45021f-f345-4752-a648-e7e3d6af0253.png">

Reply:
I felt like `appsilon.python_install` is not a mandatory role - `appsilon.ansible-rstudio-connect` doesn't strictly depend on it. 

Someone might want to install just Connect without any Python support. In such case if `appsilon.python_install` is specified in `meta/main.yml`, then it would be additional unnecessary dependency to download and install.